### ISSUE TITLE
[JOSS] Negative Coefficient of Determination in Quickstarter notebook

### ISSUE THREAD

**Issue by euronion** [IMMM-SFA/tell#34]
When running the quickstarter notebook I get negative R2 Scores for the BAs: SEC, WACM .
Can you confirm this is unintended?

See screenshot below:

![image](https://user-images.githubusercontent.com/42553970/177161396-2dd80cc0-b7fd-4fd1-b896-a438eed78f07.png)

**Comment by mcgrathc** [comment]
We were actually surprised by this ourselves. The coefficient of determination metric was calculated using sklearn’s r2_score function. According to their documentation, the value can actually be negative in cases where the fit is particularly poor. Their explanation can be found on the [documentation for the r2_score function](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.r2_score.html). We’ve added links to the documentation for the statistical metrics we used in the Jupyter notebooks where we explore the performance of the TELL MLP models.

**Comment by euronion** [comment]
Suprising and interesting indeed. Thanks!

If you add the links to the documentation then I consider the information provided to potential users sufficient to decide on the data quality and what to use which data for.

Minor note:

> Because the empirical models that underpin tell are so critically important we created a separate analysis notebook w[here](https://github.com/IMMM-SFA/tell/blob/final_mlp_updates/notebooks/tell_mlp_calibration_evaluation.ipynb) users can explore the model’s performance characteristics collectively and for individual BAs. The MLP calibration and evaluation notebook can be found [here](https://github.com/IMMM-SFA/tell/blob/final_mlp_updates/notebooks/tell_mlp_calibration_evaluation.ipynb).

From : https://immm-sfa.github.io/tell/user_guide.html

The link behind "here" is broken and leads to an 404.


Since the original point raised is not a bug, you can safely close here after the 404 has been corrected.
